db/seeds.rb

```diff
@@ -8,6 +8,7 @@
                     fetch(Rails.env, {})
 
 service_providers.each do |issuer, config|
+  next if Figaro.env.chef_env == 'prod' && config['live'] != 'true'
```

Comment:
do we need to do the `chef_env` check? why not `next` if `config['live'] != 'true'` in all envs?